ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml

```diff
@@ -17,21 +21,62 @@
 
     <input type="hidden" name="type" value="@((int)Model.Type)"/>
     <input type="hidden" name="affectedId" value="@Model.AffectedId"/>
+    
+    <div class="ui yellow segment">
+        <div><b>Case Type:</b> @Model.Type.ToString()</div>
+        <div><b>Affected User:</b> <a href="/user/@Model.AffectedId">@Model.AffectedUser!.Username (@Model.AffectedId)</a></div>
+    </div>
+    
+    <div class="ui yellow segment">
+        <div class="field">
+            <label style="text-align: left" for="reason">Reason</label>
+            <textarea name="reason" id="reason" spellcheck="true" rows="1"></textarea>
+        </div>
+        <div class="field">
+            <label style="text-align: left" for="mod-notes">Mod Notes</label>
+            <textarea name="modNotes" id="mod-notes" spellcheck="true" rows="1"></textarea>
+        </div>
+        <div class="field">
+            <label style="text-align: left" for="expires">Expiration</label>
+            <input type="datetime-local" name="expires" id="expires" required />
```

Comment:
Are timezones being appropriately handled here, or at the point that it hits the DB

Reply:
This is how the old form handled it so I'm not going to touch it. It does work correctly from what I've seen moderating beacon.

Reply:
ok